catalog/core/catalog-core-impl/filter-proxy/src/main/java/ddf/catalog/filter/proxy/adapter/GeotoolsFilterAdapterImpl.java

```diff
@@ -703,7 +705,7 @@ public Object visit(Beyond filter, Object delegate) {
     ExpressionValues filterValues = getExpressions(filter, delegate);
     String wkt = geometryToWkt(filterValues.literal);
 
-    if (distance != 0) {
+    if (abs(distance) > .000001) {
```

Comment:
❓ how did you come up with this number for the distance. Is it worth a comment in the code? Looks like a magic number.

Reply:
It kind of is a magic number. I'll go through and add comments describing what it is.